metadata/TMAnalyzerModel__21__metadata.json

```diff
@@ -1,24 +1,24 @@
-{
-    "name": "TMAnalyzerModel",
-    "retrainable": true,
-    "gpu": 1,
-    "processorType": "CPU",
-    "description": "Task Mining Model package meant to process and produce the data containing detected tasks.",
-    "inputDescription": "Dataset that contains files captured and uploaded by Task Mining Desktop application (or uploaded via Task Mining Uploader tool).",
-    "outputDescription": "Definition of the tasks that can be visualized in Task Mining Admin Console. Data will be imported into Admin Console Discovered Tasks section automatically some time after the package is finished.",
-    "changeLog": "Internal Version 1.1.5",
-    "cpu": 0,
-    "inputType": "JSON",
-    "displayName": "TMAnalyzerModel",
-    "memory": 0,
-    "mlPackageLanguage": "PYTHON38",
-    "projectId": "[project-id]",
-    "stagingUri": "[staging-uri]",
-    "projectName": "UiPath Task Mining",
-    "projectDescription": "UiPath Task Mining models to analyze captured data",
-    "tenantName": "UiPath",
-    "minAIFabricVersion": "v21.10",
-    "languageVersion": 0,
-    "version": 2,
-    "contentUri": "https://<placeholder-url>/publicmodels/AIC/TaskMining/2/task_mining.zip"
-}
+{
+    "name": "TMAnalyzerModel",
+    "retrainable": true,
+    "gpu": 1,
+    "processorType": "CPU",
+    "description": "Task Mining Model package meant to process and produce the data containing detected tasks.",
+    "inputDescription": "Dataset that contains files captured and uploaded by Task Mining Desktop application (or uploaded via Task Mining Uploader tool).",
+    "outputDescription": "Definition of the tasks that can be visualized in Task Mining Admin Console. Data will be imported into Admin Console Discovered Tasks section automatically some time after the package is finished.",
+    "changeLog": "Internal Version 1.1.6",
+    "cpu": 0,
+    "inputType": "JSON",
+    "displayName": "TMAnalyzerModel",
+    "memory": 0,
+    "mlPackageLanguage": "PYTHON38",
+    "projectId": "[project-id]",
+    "stagingUri": "[staging-uri]",
+    "projectName": "UiPath Task Mining",
+    "projectDescription": "UiPath Task Mining models to analyze captured data",
+    "tenantName": "UiPath",
+    "minAIFabricVersion": "v21.10",
+    "languageVersion": 0,
+    "version": 21,
```

Comment:
@vivekjhauipath @vpn87  Can you guys confirm if this kind of jump is ok to have w.r.t OOB installer.

Reply:
Please let me know if that is not recommended - will change the version to `2`

Reply:
It will work, aesthetics I can't comment on